src/Trigger/RabbitMQTriggerAttribute.cs

```diff
@@ -60,5 +60,15 @@ public RabbitMQTriggerAttribute(string hostName, string userNameSetting, string
         /// Gets or sets the Port used. Defaults to 0.
         /// </summary>
         public int Port { get; set; }
+
+        /// <summary>
+        /// Enable or disable ssl in RabbitMQ connection.
+        /// </summary>
+        public bool Ssl { get; set; }
+
+        /// <summary>
+        /// Enable os disable checking certificate when Ssl is enabled (not recommended for production).

```

Comment:
It would be better to have more descriptive attribute properties like `EnableSsl` and `SkipCertificateValidation`.

Reply:
Found #164 which is accomplishing same stuff. Can you check that one and see if `bool insecureSsl` can be replaced with `SslPolicyErrors acceptablePolicyErrors` so user has freedom of which errors to allow?

Reply:
According to https://docs.microsoft.com/es-es/dotnet/api/system.net.security.sslpolicyerrors , the interesting errors when skipping certificate validation are RemoteCertificateChainErrors and RemoteCertificateNameMismatch and only for development or testing purposes. So, in my opinion they have to be together.

The RemoteCertificateNotAvailable never should be accepted, as it will always produce an exception when trying to establish a SSL session.

For these reasons, in my opinion, it is better setting the attribute/option as a boolean to simplify code. Those who activate SkipCertificateValidation have to know what they are doing and do it under their own risk.

Do you agree?